From 065dbb06fdc2b1f74459fcde6b6e5092ac5c68a4 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Sun, 1 Feb 2015 01:12:49 +0000
Subject: [PATCH] Makefile.am: remove ACLOCAL_AMFLAGS=-I libltdl/m4

Fix for new libtool:
libtoolize:   error: AC_CONFIG_MACRO_DIRS([m4]) conflicts with ACLOCAL_AMFLAGS=-I libltdl/m4.
autoreconf: libtoolize failed with exit status: 1

Upstream-Status: Pending

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 Makefile.am | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 58dc567..11bea75 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,5 +1,3 @@
-ACLOCAL_AMFLAGS=-I libltdl/m4
-
 SUBDIRS = \
 	extras \
 	log \
-- 
2.0.1

